From 2060b2829ccec12f0920c4124877e641d49d8b64 Mon Sep 17 00:00:00 2001
From: Alexander Martinz <amartinz@shiftphones.com>
Date: Wed, 30 Jan 2019 22:13:39 +0100
Subject: [PATCH 02/20] Eleven: remove junit dependency

We are just pulling in junit for an runtime assertion call
within a method, which is even unused.

Remove junit call and dependency but leave the unused method there
which will be rm -rf'ed in followup changes.

Change-Id: Ib71ac78f3e0bbabcb5b4acdaa34e853e39b616aa
Signed-off-by: Alexander Martinz <amartinz@shiftphones.com>
---
 Android.mk                                               | 3 +--
 src/org/lineageos/eleven/widgets/LetterTileDrawable.java | 3 ---
 2 files changed, 1 insertion(+), 5 deletions(-)

diff --git a/Android.mk b/Android.mk
index 754a2aa..391067b 100644
--- a/Android.mk
+++ b/Android.mk
@@ -11,8 +11,7 @@ LOCAL_RESOURCE_DIR := $(addprefix $(LOCAL_PATH)/, res)
 LOCAL_STATIC_JAVA_LIBRARIES := \
     android-support-v8-renderscript \
     android-common \
-    guava \
-    junit
+    guava
 
 LOCAL_STATIC_ANDROID_LIBRARIES := \
     android-support-v4 \
diff --git a/src/org/lineageos/eleven/widgets/LetterTileDrawable.java b/src/org/lineageos/eleven/widgets/LetterTileDrawable.java
index fdbc8bd..65fdc28 100644
--- a/src/org/lineageos/eleven/widgets/LetterTileDrawable.java
+++ b/src/org/lineageos/eleven/widgets/LetterTileDrawable.java
@@ -30,8 +30,6 @@ import android.graphics.Typeface;
 import android.graphics.drawable.Drawable;
 import android.text.TextUtils;
 
-import junit.framework.Assert;
-
 import org.lineageos.eleven.R;
 import org.lineageos.eleven.cache.ImageWorker.ImageType;
 import org.lineageos.eleven.utils.BitmapWithColors;
@@ -278,7 +276,6 @@ public class LetterTileDrawable extends Drawable {
      * The default is 0.0f.
      */
     public void setOffset(float offset) {
-        Assert.assertTrue(offset >= -0.5f && offset <= 0.5f);
         mOffset = offset;
     }
 
-- 
2.17.1

